### DISCUSSION TITLE
Overrides aren't working

### DISCUSSION THREAD

**Discussion by Mitovo** [GetPublii/Publii#1858] [Help]
Hello,

(My title should read "Need help with overrides"; couldn't update it :p)

So, I went through the documentation on how to use Overrides. Specifically, I want to override the visual styles for my website.
I'm using the N-Tech theme.

Initially, I made what looks like a pretty common mistake, updating the style.css. This didn't work, of course.
I looked it up and found that style.css, and others, are all 'combined' into main.css and that's the one I should be editing.
I double-checked the folder structure to make sure I have that correct, and I do:
C:\Mike\Documents\Publii\sites\"sitename"\input\themes\n-tech\n-tech-override\assets\css\

So, I edited main.css - starting simple. I just altered the body background color, from "FFFFFF" to "00FF00", to turn it green.
Saved the main.css. Preview the site... background is still white.

Tried changing other styles, and the same result. Colors aren't reflecting my modifications.

So... I'm at a loss with how this works?

Thanks for any assistance!

**Comment by shawngiese** [comment]
Did you try just using Tools and Plugins > Custom CSS? Maybe you do not need to go the full override way of doing things.

**Comment by candidexmedia** [comment]
The path should be:

`C:\Mike\Documents\Publii\sites"sitename"\input\themes\n-tech-override\assets\css\`

The override folder should be at the same level as the regular theme folder, not nested inside.

Otherwise, you're right about editing `main.css` 😊
